MobileApplication/MobileApplication/Views/AboutPage.xaml.cs

```diff
@@ -101,16 +104,21 @@ void PresentBudgetChart(IEnumerable<Category> categories)
                 LabelMode = LabelMode.RightOnly
             };
         }
-        void PresentTransactionChart(IEnumerable<Transaction> transactions)
+void PresentTransactionChart(IEnumerable<Transaction> transactions)
```

Comment:
Try to fix basic formating (in this case it's a tabulation) before making a commit.

Reply:
okay